Test_getLatestReleaseTage fails on Travis

It seems to occur quite frequently
```
--- FAIL: Test_getLatestReleaseTag (1.36s)
	notify_test.go:18: error getting latest release TagName via API
```

This is the problem 
```
--- FAIL: Test_getLatestReleaseTag (0.76s)
	notify_test.go:18: error getting latest release TagName via API, error: GET https://api.github.com/repos/redhat-developer/odo/releases/latest: 403 API rate limit exceeded for 35.184.96.71. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.) [rate reset in 19m23s]
```
We need to remove the GitHub call

--- a/pkg/notify/notify_test.go
+++ b/pkg/notify/notify_test.go
@@ -1,30 +1,19 @@
 package notify
 
 import (
-	"context"
 	"fmt"
-	"github.com/google/go-github/github"
-	"reflect"
+	"github.com/blang/semver"
+	"strings"
 	"testing"
 )
 
 func Test_getLatestReleaseTag(t *testing.T) {
-	client := github.NewClient(nil)
-	release, response, err := client.Repositories.GetLatestRelease(context.Background(), "redhat-developer", "odo")
-	if response != nil {
-		defer response.Body.Close()
-	}
-	if err != nil {
-		t.Errorf("error getting latest release TagName via API, error: %v", err)
-		return
-	}
-
 	tests := []struct {
 		name    string
 		success bool
 	}{
 		{
-			name:    "Fetch version from release page using API and compare",
+			name:    "parse version string and see if it returns a validated Version",
 			success: true,
 		},
 	}
@@ -32,10 +21,13 @@ func Test_getLatestReleaseTag(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			releaseTag, err := getLatestReleaseTag()
+
 			if test.success == true && err == nil {
-				if !reflect.DeepEqual(fmt.Sprintf("v%s", releaseTag), *release.TagName) {
-					t.Errorf("Expected value is %s, got %s", releaseTag, *release.TagName)
+				Semver, err := semver.Make(strings.TrimPrefix(releaseTag, "v"))
+				if err != nil {
+					t.Errorf("unable to make semver from the latest release tag: %v", releaseTag)
 				}
+				t.Log(fmt.Sprintf("getLatestReleaseTag() returned %v\n", Semver))
 			}
 		})
 	}

--- a/pkg/notify/notify_test.go
+++ b/pkg/notify/notify_test.go
@@ -15,7 +15,7 @@ func Test_getLatestReleaseTag(t *testing.T) {
 		defer response.Body.Close()
 	}
 	if err != nil {
-		t.Errorf("error getting latest release TagName via API")
+		t.Errorf("error getting latest release TagName via API, error: %v", err)
 		return
 	}
 
@@ -31,8 +31,8 @@ func Test_getLatestReleaseTag(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			releaseTag, _ := getLatestReleaseTag()
-			if test.success == true {
+			releaseTag, err := getLatestReleaseTag()
+			if test.success == true && err == nil {
 				if !reflect.DeepEqual(fmt.Sprintf("v%s", releaseTag), *release.TagName) {
 					t.Errorf("Expected value is %s, got %s", releaseTag, *release.TagName)
 				}

